lib/rdoc/generator/template/darkfish/css/rdoc.css

```diff
@@ -219,7 +219,8 @@ nav a:hover {
 #navigation-toggle {
   z-index: 1000;
   font-size: 2em;
-  position: absolute;
+  display: block;
+  position: fixed;
```

Comment:
Non-blocking: I actually like the sidebar on the left, even if it moves. I think most websites work like that.

Reply:
Sidebar is still on the left.

Reply:
Sorry. I meant the icon.

Reply:
In any case, we could add a background color to the icon, so when it is on top of text it is still visible. I think I had that in the first PR